.github/workflows/build-pipelines.yml

```diff
@@ -22,7 +22,12 @@ jobs:
       - name: Create release branch
         run: |
           git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-          git fetch --all
+          git fetch -v
+          git branch | grep release
```

Comment:
question: Does this also detect other branches with the word release in it, rather than just complete branch names that match release?

Can we just always run `git push origin -d release` without the check?

Reply:
If you do want to match exactly, you can use `git branch --list release` (you need to explicitly call with wildcards to match more)

Reply:
Actually yes, we're going to run this without the check. It will fail (exit != 0), it could be either the release branch doesn't exist, or, I assume much more rarely,  for some other environmental reason.

Considering that failure to properly delete would not happen often, I'm thinking it's a good compromise.

Reply:
I assume this works and the shell just ignores the return code of the delete request and continues running commands resulting in the action returning the status of the last command run. If you did want to fail the action if that delete failed you would have to check the status and call an explicit `exit 1` or some such.

I approved the changes which should allow you to merge this once the conversation is resolved. Assuming my reading of this is correct you can resolve the conversion and do so.

Reply:
Yes, correct. I've replaced the direct call with a wrapper that always returns 0 but stores the command exit code separately. So if we wanted to know what it was, we can.

In most cases we do want to ignore the error here and continue.